install_config/aggregate_logging.adoc

```diff
@@ -806,63 +806,6 @@ xref:fluentd-upgrade-source[Updating Fluentd's Log Source After a Docker Log
 Driver Update] for more information.
 ====
 
-[[fluentd-log-external-elasticsearch]]
```

Comment:
I think instead of removing the whole things we should indicate that the installer will fail if the variable is specified.

Reply:
Keep section header and add something like 'ES_COPY' feature is no longer supported and the installer will fail if vars are present in inventory?